Update for grammar (#962)
preactjs/preact-www: content/en/guide/v10/hooks.md

# Hooks

> Hooks are a new concept that allow you to compose state and side effects. They allow you to reuse stateful logic between components.

Reviewer: ```suggestion
The Hooks API is a new concept that allows you to compose state and side effects. Hooks allow you to reuse stateful logic between components.
```

Maybe this?
